spec.html

```diff
@@ -352,7 +358,13 @@ <h3>Notes</h3>
       type uses delta and delta-of-delta compression and
       run-length-encoding for efficient element storage. Also has an
       encoding for sparse arrays containing missing values.</li>
-    <li>The BSON "binary" or "BinData" datatype is used to represent
+    <li>Vector - A densely packed array of numbers, all of the same
+      type. This subtype supports data types that are related to the <a
+        href="https://www.mongodb.com/docs/atlas/atlas-vector-search/vector-search-overview/">Atlas
+        Vector Search</a> feature, such as <code>uint1</code>,
+      <code>int8</code>, and <code>f32</code>.
```

Comment:
I suggest avoiding motivating this. Densely packed vector data is useful in many many scientific computing fields.

Reply:
agreed, we can drop the entire "related to" sentence